packages/office-ui-fabric-react/src/components/TextField/docs/new/TextFieldOverview.md

```diff
@@ -0,0 +1,5 @@
+Text fields give people a way to enter and edit text. They’re used in forms, modal dialogs, tables, and other surfaces where text input is required.
```

Comment:
We should standardize on the use of `TextField` or `Text field` in our docs.

Reply:
It's not fully reflected in this PR yet, but the style is *probably* going to switch to using the friendly control names ("text field") instead of the code names (`TextField`) in these documentation sections. The code name would still be mentioned at the start of the article and used in the text where necessary, like in more technical sections or when it's necessary to distinguish our implementation's behavior from a generic text field.

Reply:
It's not fully reflected in this PR yet, but the style is probably going to switch to using the friendly control names ("text field") instead of the code names (TextField) in these documentation sections. The code name would still be mentioned at the start of the article and used in the text where necessary, like in more technical sections or when it's necessary to distinguish our implementation's behavior from a generic text field.